source/includes/_feature-deriving.md

```diff
@@ -395,4 +395,158 @@ POST'ing to the private variables catalog a Shoji Entity containing a ZCL functi
 
 ### Other transformations
 
```

Comment:
Fair enough about the boolean ones for now, but `{"function": "+", "args":[ {"variable":id}, {"value": 5}]}` returns a column, doesn't it?